types/jsbn/index.d.ts

```diff
@@ -1,9 +1,9 @@
-// Type definitions for jsbn v1.2
+// Type definitions for jsbn v1.3.0
 // Project: http://www-cs-students.stanford.edu/%7Etjw/jsbn/
 // Definitions by: Eugene Chernyshov <https://github.com/Evgenus>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare namespace jsbn {
+declare module 'jsbn' {
```

Comment:
It looks like jsbn supports CommonJS (when it detects an `exports` object) and a global namespace (when there is no `exports` object).

A better approach would be to remove the `declare module 'jsbn' {` entriely (making this file into an external module), and adding an `export as namespace jsbn;` line to the file. That will allow it to work in both CommonJS and globally.

Reply:
If you do this, you should also export the `RandomGenerator` and `Reduction` interfaces

Reply:
Will give it a try then test.  Schedule currently full ... please be patient.